src/database/items.lua

```diff
@@ -328,7 +328,7 @@ local function _itemData()
     missileHell = {
       name = "Missile Expansion",
       script = "<EVE0035",
-      attributes = {"missileLauncher", "nonProgressive"}
+      attributes = {"weapon", "missileLauncher", "nonProgressive"}
```

Comment:
are these `weapon` attributes used anywhere?

Reply:
They're used in the requirements for the Kulala chest, the Lab I capsule, and the dragon chest in ruined Egg Corridor.

Reply:
oh lol I'm just dumb